lib/range.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const rangeParser = require('parse-numeric-range');
+const union = require('array-union');
+
+function parseRange(files) {
+	const any = new Set();
```

Comment:
@nov parseRange should work on both ranged paths and normal paths, imagine we have an input like `['a.js', 'a.js:4']` in this case the `a.js` path has a higher priority and should cancel out `a.js`, this set is used to store file paths that do not have a range syntax in them.

Reply:
I see. Should that maybe print an error instead? Or perhaps we could run only the selected lines and print a warning "ignoring other tests from a.js since you selected lines" (well not quite that, but you get the idea).